front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -81,7 +81,7 @@ const IdeasNewPage = () => {
         context: {
           type: 'phase',
           action: 'posting_idea',
-          id: phase_id || getCurrentPhase(phases?.data)?.id || '',
```

Comment:
Our types indicate phase_id is always defined. It's a brittle construction, because we use `as` for this. However, at least at this time, we add manually add the phase id in all the places where we link to /ideas/new or /surveys/new. I feel this needs to be revisited though, because it seems like a matter of time before we forget to add in a spot.

Reply:
I've added this to the FE CDM agenda, but feel free to already have a look at this before if you have the time.